### DISCUSSION TITLE
Add a question to propose unselecting ingredient image if no ingredients are detected

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/robotoff#1192] [Ideas]
Propose to unselect ingredient pictures if no ingredient is recognized and nutrition keywords are detected.

``` It looks like the image selected for ingredients contains only nutrition information. Should we unselect it ?```

**Comment by alexgarel** [comment]
I'm not sure how you intend to do this @teolemon 

Ingredients are parsed on productopener side, so it's more product opener that could tell that. 

Or do you intend robotoff to make a mongodb request to get ingredients after product update ?

**Comment by teolemon** [comment]
Ocr, fasttext on the ingredients taxonomy
